### PR TITLE
Fix Upsample ONNX Symbolic

### PR DESCRIPTION
**Pull request by houseroad**
Adjust the change to changes in ATen

### CODE REVIEW

**Review by ailzhang — APPROVED** [review]
Thanks for fixing this!

File: torch/onnx/symbolic.py [hunk 1]
```diff
@@ -487,9 +487,11 @@ def replication_pad(g, input, padding):
 replication_pad3d = replication_pad
 
 
-def upsample_nearest2d(g, input, scale_factor):
-    return g.op("Upsample", input, width_scale_f=scale_factor,
-                height_scale_f=scale_factor, mode_s="nearest")
+def upsample_nearest2d(g, input, output_size):
+    return g.op("Upsample", input,
+                height_scale_f=float(output_size[-2]) / input.type().sizes()[-2],
+                weight_scale_f=float(output_size[-1]) / input.type().sizes()[-1],
```
Comment (vedanuj) [inline torch/onnx/symbolic.py]: This should be `width_scale_f` instead of `weight_scale_f`

Comment (houseroad) [reply torch/onnx/symbolic.py]: oops, my bad, fixed

**Pull request closed without merging**